deploy/clowdapp.yaml

```diff
@@ -314,8 +314,12 @@ parameters:
   value: INFO
 - name: CPU_LIMIT
   value: 500m
+- name: CPU_REQUEST
```

Comment:
What if we need to increase the CPU or memory one of component without changing the others? Should we have more parameters?